### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by BenedictSt**
Fixed some typos.

### CODE REVIEW

**Comment by tomerd** [comment]
thanks @BenedictSt!

**Comment by tomerd** [comment]
@swift-ci please smoke test

**Review by neonichu — APPROVED** [review]

**Comment by tomerd** [comment]
@BenedictSt could you rebase your changes? some of the code moved around :(

**Comment by BenedictSt** [comment]
No problem,
I'll do it tomorrow.

File: Tests/PackageDescriptionTests/VersionTests.swift [hunk 1]
```diff
@@ -531,7 +531,7 @@ class VersionTests: XCTestCase {
         
     }
     
-    // A semantic version string is always longer than 1 character, so the only way to test these 2 initialisers is by calling them directly.
+    // A semantic version string is always longer than 1 character, so the only way to test these 2 initializers is by calling them directly.
```
Comment (abertelrud) [inline Tests/PackageDescriptionTests/VersionTests.swift:534]: Technically not a misspelling, but yes, American spelling is the standard for code.  :)

**Review by abertelrud — APPROVED** [review]
Thanks!  There were a lot of these!  :)

**Comment by tomerd** [comment]
@swift-ci please smoke test

**Comment by tomerd** [comment]
@swift-ci please smoke test

**Comment by tomerd** [comment]
@swift-ci please smoke test linux

**Comment by tomerd** [comment]
thanks again @BenedictSt

**Pull request merged**